docs/pages/guides/authentication.md

```diff
@@ -1186,8 +1186,8 @@ export default function App() {
 
       /* @info Create a Google credential with the <code>id_token</code> */
       const auth = getAuth();
-      const provider = new GoogleAuthProvider();
-      const credential = provider.credential(id_token);
+      
```

Comment:
```suggestion
```

Thank you @betomoedano, for updating the example with a fix 🙏 

Just a small suggestion to remove the extra line between the two.